### DISCUSSION TITLE
如何在公式里加粗字母？\boldsymbol{}、\boldmath{}都不管用。。。

### DISCUSSION THREAD

**Discussion by Linkzhuu** [nju-lug/NJUThesis#89] [Q&A] [answered]
使用$\boldsymbol{m}$有如下报错
Missing character: There is no 𝑚 ("1D45A) in font cmr12!

**Comment by atxy-blip** [comment]
本来以为 <https://github.com/nju-lug/NJUThesis/issues/51> 已经解决这个问题了，实际上跟 `bm` 等宏包还是有兼容问题。我最近比较忙，一两周后再修吧。

另：这条算 bug，我转移到 issue 区。

**Comment by zepinglee** [comment, marked as answer]
`unicode-math` 的用法是 `\symbf`，还可以设置 `\unimathsetup{bold-style = TeX / ISO}` 选择粗正或粗斜。

**Reply by Linkzhuu** [reply]
感谢～
添加

> \documentclass[bold-style=ISO]{njuthesis}

已解决问题

**Reply by zepinglee** [reply]
这个最好设置 `\unimathsetup{bold-style = ISO}`，而不是加到 `\documentclass` 的参数中。

**Reply by dreistein543** [reply]
其实要具体区分一下的

> `\symbf`        加粗 非斜体
> `\symbfit`      加粗 斜体
> `\mathbfcal`  加粗 \mathcal 字符

**Reply by zepinglee** [reply]
这样要用 `\unimathsetup` 设置的。
<img width="362" alt="Screen Shot 2022-04-14 at 20 08 19" src="https://user-images.githubusercontent.com/12290822/163387692-d3ef7ebc-2873-44a2-a1ca-e63c2ebcbc3c.png">

**Reply by dreistein543** [reply]
我尝试过，目前的版本似乎设置ISO 无效（\symbf 保持非斜体），设置Tex 报错（报错信息显示说已经被fixed ）
最终我是查了其他资料整理出的上述三种加粗，某种程度上说我觉得这样更加灵活，能根据需要加粗斜体非斜体
